spec/registrant_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+
+RSpec.describe Registrant do
+  before(:each) do
+
+    @registrant_1 = Registrant.new('Bruce', 18, true )
+    # pry(main)> registrant_1 = Registrant.new('Bruce', 18, true )
+    # #=> #<Registrant:0x000000015c10bed8 @age=18, @license_data={:written=>false, :license=>false, :renewed=>false}, @name="Bruce", @permit=true>
```

Comment:
For now, if leaving these comments in the project helps you, go ahead, but if you're ever doing a project that you're showing off in github or showing off in a job interview, you'll want to remove comments like this in the final code.